products/ibm-aix.md

```diff
@@ -0,0 +1,91 @@
+---
+title: IBM AIX
+category: os
+iconSlug: ibm
+permalink: /ibm-aix
+versionCommand: oslevel -s
+releasePolicyLink: https://www.ibm.com/support/pages/standard-and-enhanced-ibm-support-lifecycle-policies
+activeSupportColumn: true
+releaseDateColumn: true
+releaseLabel: >-
+  {{'__RELEASE_CYCLE__' | split:'.' | pop:2 | join(".")}} TL{{'__RELEASE_CYCLE__' | split:'.' | last}}
+
+auto:
+-   custom: true
+
+releases:
+-   releaseCycle: "7.3.1"
+    releaseDate: 2022-12-01
+    support: 2025-12-31
+    eol: 2025-12-31
+    latestReleaseDate: 2023-04-28
+    link: https://www.ibm.com/docs/en/aix/7.3?topic=notes-aix-731-release
+
+-   releaseCycle: "7.3.0"
+    releaseDate: 2021-12-01
+    support: 2024-12-31
+    eol: 2024-12-31
+    latestReleaseDate: 2023-03-10
+    link: https://www.ibm.com/docs/en/aix/7.3?topic=notes-aix-73-release
```

Comment:
Looks like a changelogTemplate could be used for most (all ?) the release cycles . Did not tested it but this should do the trick:

```
changelogTemplate: https://www.ibm.com/docs/en/aix/{{'__LATEST__'|split:'.'|pop|join('')}}?topic=notes-aix-{{'__LATEST__'|split:'.'|join('')}}-release
```